targets/csharp/source/PlayFabSDK/source/PlayFabSettings.cs.ejs

```diff
@@ -37,6 +37,8 @@ namespace PlayFab
         public static string TitleId { get { return staticSettings.TitleId; } set { staticSettings.TitleId = value; } }
         #endregion Deprecated staticSettingsredirect properties
 
+        public static string ConnectionString { get { return staticSettings.ConnectionString; } set { staticSettings.ConnectionString = value; } }
```

Comment:
Given the Obsolete attributes above, should we be adding this?  It sounds like we developers should just be referencing staticSettings instead of this property directly.